bin/wp-cli.php

```diff
@@ -82,6 +82,31 @@ public function flush() {
 		}
 	}
 
+	/**
+	 * Add date range when retrieving posts in bulk.
+	 * Dates need to be passed as mmddyyy. See synopsis for index function.
+	 *
+	 * @param $args array
+	 * @return $args array
+	 */
+	public function apply_date_range( $args ) {
```

Comment:
You're right. WP-CLI sees apply_date_range as a subcommand and if I make it private the add_filter call won't work. I haven't found a way to tell phpdoc to ignore a public method.

Jetpack works around this by putting a function outside the class (https://github.com/Automattic/jetpack/blob/master/class.jetpack-cli.php#L573) but in this case I'm using add_filter and by using a function instead of a method I lose the ability to pass $this->date_range.

I could pass the date range with a global variable but I don't feel that's too elegant, see https://github.com/alleyinteractive/searchpress/commit/a93e4c2d3800ad6fd61f0db9cd898b3672e70d72. I'm open to suggestions.